Procfile

```diff
@@ -1,5 +1,5 @@
 # Procfile for development using the new threaded worker (scheduler, twitter stream and delayed job)
-web: bundle exec rails server
+web: bundle exec rails server -b0.0.0.0
```

Comment:
Interesting, I always thought you would need a space between the switch and the argument but it seems to work fine that way :smile: